crates/class-groups-constants/src/lib.rs

```diff
@@ -7,11 +7,29 @@ use class_groups::{
 };
 use constants::{DECRYPTION_KEY_SHARE_PUBLIC_PARAMETERS, DYCRPTION_SHARES, NETWORK_DKG_OUTPUT};
 use group::{secp256k1, PartyID};
+use serde::{Deserialize, Serialize};
 use std::collections::HashMap;
 use twopc_mpc::secp256k1::class_groups::{
     ProtocolPublicParameters, FUNDAMENTAL_DISCRIMINANT_LIMBS, NON_FUNDAMENTAL_DISCRIMINANT_LIMBS,
 };
 
+/// Contains the public keys of the DWallet.
+///
+/// Being used to sign on with the Sui signature key when encrypting this DWallet to another user. The receiving user
+/// can later verify the signature is valid and know this DWallet decentralized output has been signed by the source
+/// address.
```

Comment:
This doc is very confusing, and I don't understand anything that it says.
These keys are the dWallet keys, it is being used for the dWallet `sign` process.
The fact that it is signed by the encryption key in the encryption transfer process is irrelevenat.
Please explain and rephrase.

Reply:
Also, why is it in class-groups-constants, move it to dWallet-constants.